Geometry/CaloTopology/src/HcalTopology.cc

```diff
@@ -823,8 +817,7 @@ int HcalTopology::decAIEta(const HcalDetId& id, HcalDetId neighbors[2]) const {
 void HcalTopology::depthBinInformation(
     HcalSubdetector subdet, int etaRing, int iphi, int zside, int& nDepthBins, int& startingBin) const {
   if (subdet == HcalBarrel) {
-    if ((mode_ == HcalTopologyMode::SLHC) || (mode_ == HcalTopologyMode::H2HE) || (mode_ == HcalTopologyMode::Run3) ||
-        (mode_ == HcalTopologyMode::Run4)) {
+    if (phase2() || (mode_ == HcalTopologyMode::H2HE)) {
```

Comment:
This does not include Run3 now: is it correct? (Just checking)

Reply:
Yes it is